config/locales/js-en.yml

```diff
@@ -1123,7 +1123,7 @@ en:
       sharing:
         share: "Share"
         title: "Share work package"
-        show_all_users: "Show all shared users"
+        show_all_users: "Show all users with whom the work packages has been shared with"
```

Comment:
According to the docs, the meaning of the translation was incorrect. Here's my proposed version.
cc: @psatyal